README.md

```diff
@@ -8,90 +8,20 @@
-- Choose a dataset from the [list of Nusantara datasets](https://github.com/orgs/IndoNLP/projects/2/views/1). 
-<p align="center">
-    <img src="./docs/_static/img/select-task.jpeg" style="width: 80%;"/>
-</p>
-
-- Assign yourself an issue by clicking the dataset in the project list, and comment `#self-assign` under the issue. **Please assign yourself to issues with no other collaborators assigned**. You should see your GitHub username associated to the issue within 1-2 minutes of making a comment.
-<p align="center">
-    <img src="./docs/_static/img/self-assign.jpg" style="width: 80%;"/>
-</p>
-
-- Search to see if the dataset exists in the 🤗 [Hub](https://huggingface.co/datasets). If it exists, please use the current implementation as the `source` and focus on implementing the [task-specific `nusantara` schema](https://github.com/IndoNLP/nusa-datasets/blob/master/task_schemas.md).
-
-**Option B: Implement a new dataset not on the list**
-
-If you have an Indonesian and local languages dataset you would like to propose in this collection, you are welcome to [make a new issue](https://github.com/IndoNLP/nusa-datasets/issues/new/choose) and fill out relevant information. **Make sure that your dataset does not exist in the 🤗 [Hub](https://huggingface.co/datasets).**
-
-If an admin approves it, then you are welcome to implement this dataset, and it will count toward contribution credit.
-
-### 2. Implement the data-loading script for your dataset and create a PR
-
-[Check out our step-by-step guide to implementing a data loader with the nusantara schema](CONTRIBUTING.md).
-
-**Please do not upload the data directly; if you have a specific question or request, [reach out to an admin](#Community-channels)**
-
-As soon as you have opened a PR, the dataset will be marked as `In Progress` in the
-[list of Indonesian datasets](https://github.com/orgs/IndoNLP/projects/2/views/1).
-When an admin accepts the PR and closes the corresponding issue, the dataset will be
-marked as `Done`.
-
-## Community channels
-
-We welcome contributions from a wide variety of backgrounds; we are more than happy to guide you through the process. For instructions on how to get involved or ask for help, check out the following options:
-
-#### Join our WhatsApp group.
-Alternatively, you can ping us on the [WhatsApp group](https://chat.whatsapp.com/Jn4nM6l3kSn3p4kJVESTwv). The WhatsApp group can be used to share information quickly or ask code-related questions.
-
-#### Make a Github Issue
-For quick questions and clarifications, you can [make an issue via Github](https://github.com/IndoNLP/nusa-datasets/issues/new/choose).
-
-You are welcome to use any of the above resources as necessary. 
+Yes! Aside from new dataset collection, we are also centralizing existing datasets in a single schema that makes it easier for researchers to use Indonesian NLP datasets. You can help us there by building dataset loader. More details about that here.
```

Comment:
"here" means we create a new .md file, move the details of the dataloader there, and ilink it here?